_life/money-americans.md

```diff
@@ -0,0 +1,406 @@
+than what you would have to pay if you filled out the _Taxation ordinaire ultérieure_
+(TOU) form, so requesting a TOU if you don't have to is not worth it. In the year
+that you move to Switzerland and hence have Swiss income for only part of the year,
+you may think that your total income falls below the minimum for taxation and you
+could get a tax refund by filing a TOU. This is not the case.[^7] Tax rates for
+"periodic" income are calculated by scaling the income up to the full year. Plus,
+even if you could get a tax refund in the first year, a TOU election cannot be
+undone, and you will end up paying more tax in following years as mentioned before.
+
+Note that you are _required_ to file a TOU under certain circumstances, one of which
+is if you have any income not subject to source withholding, such as investment
+returns. This TOU is not "sticky" like the elective TOU. Then there are two or maybe
+three relevant annexes to the standard tax return:
+
+ 1. Annex 1, _État des titres et autres placements de capitaux_, or Statement of
+    securities and other capital investments. Most of it will be blank except for the
+    number(s) copied over from the annex(es) below. (Except if you have Swiss
+    investments, of course, which you shouldn't.)
+
+ 1. Annex DA-1, _Demande d’imputation d’impôts étrangers prélevés à la source pour
+    dividendes et intérêts étrangers échus en 2022_, or Request for imputation of
+    foreign taxes already levied at the source on foreign dividends and interest
+    issued in 2022. Here I believe you can report any US taxes withheld on your
+    investments in the US. If the tax administration accepts this, then they will
+    refund you that amount from the Swiss taxes you paid/will pay on the same income
+    to avoid double taxation. The income will still count towards your total income
+    for the purposes of determining tax _rates_.
+
+	What I'm not sure about is if you have foreign investments held in the US, and
+	your US broker pays the relevant foreign taxes for you, and reports the amount so
+	that you can use it as foreign tax credit in your US returns. Can you report this
+	amount in DA-1? Morally I feel you should be able to, to avoid double taxation,
+	but I can't be sure about the legal answer.
+
+ 1. Annex R-US 164, _Demande en remboursement de la retenue supplémentaire d’impôt
+    USA 2022_, or Request for refund of US supplemental withholding tax 2022. This is
+    where it gets tricky. I believe this is for US investments made through _Swiss_
+    brokers, who automatically withhold some tax as a compliance measure (maybe
+    something about FATCA and the vagaries of the US-Swiss tax treaties?), so that
+    you have to file a return to get it back. I don't think this applies for US
+    investments made in the US.
+
+Lastly, two points worth clarifying. One, that the Canton of Vaud's tax return filing
+deadline is March&nbsp;15, a month before your US federal tax return is due. Two,
```

Comment:
Note that in practice the deadline is more like June, though this technically can't be relied upon, it has been that way since forever. If you email them the first line of the auto-reply is a formal version of "don't worry about the deadline, everyone gets extra time til June automatically".